### PR TITLE
Adding the LPR example to the software cookbook

### PR DESCRIPTION
**Pull request by SanggyuChong**
Hello all,

In this PR, we have re-vamped and simplified one of the notebooks coming from the original LPR publication for the software cookbook. It's a simple yet straightforward example of how one can go from structures to computing the SOAP descriptors, then performing the LPR analysis, and studying the changes before and after dataset modification.

Would be nice to get this merged in. Let us know of any changes you would like to request. (Co-authored by @fgrassel)

### CODE REVIEW

**Comment by SanggyuChong** [comment]
Pinning @Luthaf to follow up on the quick chat about updating the flake8 config to bypass E203 error with the slices.

File: examples/lpr/lpr.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
+"""
+LPR analysis for amorphous silicon dataset
+==========================================
+
+.. start-body
```
Comment (Luthaf) [inline examples/lpr/lpr.py]: This is not needed! 

Can you add the authors for this example, like we do for the other ones here https://github.com/lab-cosmo/software-cookbook/pull/46?

**Comment by DavideTisi** [comment]
I did a rebase to master.
I looked at the folder it seems very good.
is it finished or is there something you want to add @SanggyuChong ?

**Comment by SanggyuChong** [comment]
from our point of view, it is finished.

**Review by DavideTisi — APPROVED** [review]

**Pull request merged**